config/services.yml

```diff
@@ -573,10 +593,7 @@ https://cmr.uat.earthdata.nasa.gov:
     umm_s:
       - S1240682712-EEDTEST
       - S1241070355-GES_DISC # GESDISC_HOSS
-    collections:
-      - C1238392622-EEDTEST  # RSSMIF16D test collection UAT
-      - C1222931739-GHRC_CLOUD  # RSSMIF16D UAT
-      - C1238621141-POCLOUD # GHRSST Level 4 MUR Global Foundation Sea Surface Temperature Analysis (v4.1)
```

Comment:
Just to verify: are you responsible for this association, or is PO.DAAC depending on it somehow?

Reply:
I couldn't remember who added this association, so I took a quick look at the git history, and it looks like Mike Gangl added this line. I've preserved the association with a UMM-S to UMM-C association in CMR (against the EEDTEST version of the UMM-S record), but maybe PODAAC might want their own copy in the future. (I'm not sure I've completely answered the question there)

The other thing to note is that when I tried a request to test the association was good, it first went to the ASF GDAL Subsetter, as the collection is also registered there, and that service is higher in the `services.yml`.

Reply:
Would you mind either re-adding the collection so PO.DAAC can manage it as they see fit or confirming with Mike that it can be removed?

Reply:
Sure - I'll check with Mike about what he wants done with this collection.